src/App.tsx

```diff
@@ -1,18 +1,60 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Pagination } from './components/Pagination';
 import './App.css';
-import { getNumbers } from './utils';
+import { getNumbers, getVisibleAmountItems } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [itemsPerPage, setItemsPerPage] = useState(10);
```

Comment:
check failed tests
<img width="509" alt="Screenshot 2023-03-14 at 17 21 42" src="https://user-images.githubusercontent.com/53228259/225049533-bcb8ffa2-1933-4dd8-8bce-97687e2cad62.png">

```suggestion
  const [itemsPerPage, setItemsPerPage] = useState(5);
```
also add types to all states